vdr/vdr.go

```diff
@@ -106,6 +139,11 @@ func (r *VDR) Start() error {
 		_, err = r.network.Reprocess(context.Background(), "application/did+json")
 	}
 
+	err = r.network.DiscoverNodes(didnuts.Finder{Store: r.store})
```

Comment:
Network is started after VDR, moment of calling in terms of startup order is the same. It was moved to here for design reasons:

To remove network's dependency on `didnuts.store.Store`, since that would require odd coding:
- It needs a dependency on an internal detail (storage) of a specific DID method (did:nuts)
- It would require initializing the did:nuts store before network (like we had before), which is odd since it's an internal of did:nuts
- Leads to an additional configuration dependency of network on VDR, while there already is the `DIDResolver`.

Now it only depends on did:nuts as sort of visitor of the `Finder`, since network wants to discover nodes from DID documents. Could be fixed by another abstraction layer (like VDR feeding an address book), but that would probably be overkill.

Reply:
A good comment line would help ;)